src/triggers.js

```diff
@@ -240,6 +241,51 @@ function logTriggerErrorBeforeHook(triggerType, className, input, auth, error) {
   });
 }
 
+export function maybeRunAfterFindTrigger(triggerType, auth, className, results, config) {
+  return new Promise((resolve, reject) => {
+    const trigger = getTrigger(className, triggerType, config.applicationId);
+    if (!trigger) {
+      return resolve();
+    }
+    const request = getRequestObject(triggerType, auth, null, null, config);
+    const response = {
+      error: function(code, message) {
+        if (!message) {
+          message = code;
+          code = Parse.Error.SCRIPT_FAILED;
+        }
+        const scriptError = new Parse.Error(code, message);
+        return reject(scriptError);
+      }
+    };
+    logTriggerSuccessBeforeHook(triggerType, className, 'AfterFind', JSON.stringify(results), auth);
+    response.results = results.map(result => {
+      //setting the class name to transform into parse object
+      result.className=className;
+      return Parse.Object.fromJSON(result);
+    });
+    const triggerPromise = trigger(request, response);
+    logTriggerAfterHook(triggerType, className, JSON.stringify(modifiedJSONResults), auth);
```

Comment:
modifiedJSONResults is not defined here...
This is probably why you can't use anything else but `var` https://github.com/ParsePlatform/parse-server/pull/2968/files#diff-86ec9a2dbbd2c72ada5a115e9f9c4e6aR281 as this variable leaks out of the scope.

This line should be put after the promise has resolved. here